src/resources/i18n/translations/en.json

```diff
@@ -260,20 +260,20 @@
   "Custom API domain": "Custom API domain",
   "Custom Feeds": "Custom Feeds",
   "Custom RSS feeds to parse in parallel": "Custom RSS feeds to parse in parallel",
-  "Custom RSS feeds to parse in parallel helper text": "Parsing too many feeds in parallel may cause issues on some devices. 3 is recommended.",
+  "Custom RSS feeds to parse in parallel helper text": "Parsing more than three feeds at once may cause issues on some devices.",
   "Custom web domain": "Web domain",
-  "Custom web domain subtext": "Custom domains to use an alternate server for your app data.",
+  "Custom web domain subtext": "These allow choice of app data server.",
   "Dark Mode": "Dark",
   "Data Settings": "Data Settings",
-  "Default downloaded episode limit for each podcast": "Max downloaded episodes for each podcast",
+  "Default downloaded episode limit for each podcast": "Limit",
   "Delete": "Delete",
   "DELETE": "Delete",
   "Delete Account": "Delete account",
   "Delete All Downloaded Episodes": "Delete all downloaded episodes",
   "Delete cache": "Delete cache",
   "Delete Clip": "Delete Clip",
-  "Delete Downloaded Episodes": "Delete downloaded episodes",
-  "Delete downloaded episodes after end is reached": "Delete downloaded episodes when played",
+  "Delete Downloaded Episodes": "Delete all",
+  "Delete downloaded episodes after end is reached": "Delete after playback",
```

Comment:
Assuming it also affects clips.

Reply:
Hmmm I think "Delete downloaded episodes" is the clearest, and anything that involves deleting as a user would make me wary and I'd like extra context. It's on the Settings > Downloads screen though so "Delete all" seems fine.

Auto-delete doesn't affect clips. Currently the feature works as follows: 1) You have a downloaded episode. 2) You play it until the end. 3) The app automatically marks the downloaded episode for deletion. 4) Next time you restart the app, the "marked for deletion" episode files are auto removed.

If you reach the end of a clip, that doesn't necessarily mean you want the episode to be deleted, so we only do this with at end of episode not end of clip.

Also, our "clips" are simply time stamps referenced within the full episode file. If we were to start chopping up and hosting clips on our servers it could make sense to download them individually and auto-delete them after finished, but that's not how it works currently.

Reply:
Both "after playback" and "when played" are a bit misleading wrt. when deletion actually happens.
The former maybe less so.
If deletion is within question, then downloading is a precursor. It also works for the cache of streamed episodes.

As for the clips, I think a "Save clip" button in clips, or "Make clip" in episodes could be a way to add stuff to the device library.